common/models/Tag_definitions.csv

```diff
@@ -192,7 +192,7 @@ hardware,Hardware,,Project Category,Project Subcategory,infrastructure
 journalism-action,Journalism Action,,Project Category,Project Subcategory,media
 media-analysis,Media and Social Media Analysis,,Project Category,Project Subcategory,media
 open-source,Open Source,,Software License,,
-propreitary,Propreitary,,Software License,Open Source License,open-source
+proprietary,Proprietary,,Software License,Open Source License,open-source

```

Comment:
If we want to change the slug 'propreitary' we'll need to add a special database migration to change all existing instances of that tag.